src/openeo_odc/map_processes_odc.py

```diff
@@ -171,7 +190,7 @@ def create_string(dict_input):
     inputs = []
     to_remove = []
     for key, value in dict_input.items():
-        if key in ('x', 'y', 'data', 'value', 'base', 'p','target'):
+        if key in ('x', 'y', 'data', 'value', 'base', 'p', 'target', 'parameters', 'function', 'process'):
```

Comment:
Can the value of `parameters`, `function` and `process` be a previous node or are the always a number? In the latter case there is no point in adding them here, this function only handles such "node parameters"

Reply:
'parameters' can not be a previous node in the fit_curve process, but can be in the predict_curve node. 'process' and 'function' can be nodes.

Reply:
And none of them can be a string or?

Reply:
correct, 'parameters' should be either a float (or int) or a list of numbers in the fit_curve process, function and process should be nodes.

Reply:
perfect :+1: